CMakeLists.txt

```diff
@@ -194,6 +194,12 @@ endif()
 message(STATUS "Compiling with Capstone")
 find_package(CAPSTONE 5 REQUIRED)
 message(STATUS "CAPSTONE version: ${CAPSTONE_VERSION}")
+if(NOT ${CS_VERSION_MAJOR} MATCHES 4)
```

Comment:
What if it's capstone 3? Maybe we should use [`GREATER_EQUAL`](https://cmake.org/cmake/help/latest/command/if.html#greater-equal)?

Reply:
Good point. Well, ok to lets keep `if(NOT ${CS_VERSION_MAJOR} MATCHES 4)` if it's better for CIs.

Reply:
Other solutions:

1. we could upgrade the cmake used by vcpkg. `GREATER_EQUAL` is used from cmake 3.7.2.
2. On cmake [3.0.2](https://cmake.org/cmake/help/v3.0/command/if.html) we have the `GREATER` variable.

Reply:
Looks like it's a cmake [3.30 used](https://github.com/JonathanSalwan/Triton/actions/runs/10040279412/job/27745998115?pr=1318#step:4:26) by vcpkg. So, using `GREATER` should be ok